templates/sales-team/sales-team.css

```diff
@@ -0,0 +1,204 @@
+.default-content-wrapper {
+    h1 {
+        text-align: center;       
+    }
+
+    strong {
+        font-weight: 900;        
+    }
+
+    p {
+        margin-top: 30px;
+        text-align: center;
+        font-size: var(--body-font-size-m);
+    }
+
+    h1:nth-of-type(2) {
+        margin-top: 30px;
+        margin-bottom: 30px;
+        color: var(--light-gray);
+        font-weight: 400;        
+    }
+
+    h1:nth-of-type(2) > strong {
+        font-weight: 900;
+        color: black;
+    }
+}
+
+
+@media screen and (min-width: 992px) {
+    .sales-team .specialists-sales-team {
+        width: 970px !important;
```

Comment:
Any reason why we need important here? This could indicate that there's a problem higher up